engine/docker/docker.go

```diff
@@ -157,10 +157,31 @@ func (e *Docker) Destroy(ctx context.Context, pipelineConfig *spec.PipelineConfi
 	containers := e.containers
 	e.mu.Unlock()
 
+	timeout := time.Second * 30 // similar to k8s default timeout in pods
```

Comment:
Could we decrease the timeout here? I'm guessing the plugin has to make a network call so maybe something like 5-10 seconds would be better?